nixos/modules/services/security/kanidm.nix

```diff
@@ -795,7 +795,7 @@ in
               attrNames oauth2Cfg.scopeMaps
             )
           ))
-          ++ (optionals (cfg.provision.extraJsonFile == null) (
+          ++ (optional (cfg.provision.extraJsonFile == null) (
```

Comment:
The `optionals` 5 lines above this line also gets passed the result of `assertGroupsKnown` like here. That should then also be wrong, right?

Reply:
Probably!